gradio/components.py

```diff
@@ -2305,13 +2715,18 @@ def __init__(
         default_value: str = " ",
         *,
         label: Optional[str] = None,
+        show_label: bool = True,
         css: Optional[Dict] = None,
+        visible: bool = True,
         **kwargs,
     ):
         """
         Parameters:
         default (str): IGNORED
-        label (str): component name in interface.
+        label (Optional[str]): component name in interface.
```

Comment:
We deprecated the component, but we are still updating it :D 

Shall we move deprecated components under another file?

Reply:
Removed.